### ISSUE TITLE
308 Redirects are always turned into a GET request

### ISSUE THREAD

**Issue by schmod** [a85/POSTMan-Chrome-Extension#4114]
If I try to make a POST, PUT, or DELETE request to an endpoint that returns a 308 redirect, Postman always transforms the into a GET.  

Here's a runkit app that demonstrates the issue:
https://runkit.com/schmod/5a6b6b89e3908200129ff6cf

Use Postman to make a PUT request to: 
https://runkit.io/schmod/5a6b6b89e3908200129ff6cf/branches/master/redirect

![screen shot 2018-01-26 at 1 03 01 pm](https://user-images.githubusercontent.com/1102667/35453760-5f8122fe-0299-11e8-883c-9c825b8af4e5.png)

cURL correctly handles the request:

```shell
$ curl -L -XPUT https://runkit.io/schmod/5a6b6b89e3908200129ff6cf/branches/master/redirect

Got a PUT request
```

MDN's [documentation for the 308 status code](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/308) indicates that Postman's behavior is non-standard, and does not mimic any common user-agent in the wild.  

---

Postman for Mac
Version 5.5.2
OS X 16.7.0 / x64

**Comment by sdnts** [comment]
Hi @schmod, we've seen this kind of behavior before, [this comment](https://github.com/postmanlabs/postman-app-support/issues/2604#issuecomment-271503172) explains why it is this way. I'm adding this as a feature request (to be able to control this behavior)

**Comment by schmod** [comment]
Thanks for getting in touch!  The previous comment correctly states why the current behavior exists when Postman handles a 302.

However, Postman's current behavior for 308s does not match any browser that I know of.

**Comment by j4velin** [comment]
Same issue for redirects via status code 307.

"This status code is similar to 302 (Found), except that it does not allow changing the request method from POST to GET."
https://tools.ietf.org/html/rfc7231#section-6.4.7

**Comment by codenirvana** [comment]
@schmod @j4velin This has been fixed in Postman native apps starting from 6.7.1.
